### DISCUSSION TITLE
MPU6050_DMP not working with Processing IDE

### DISCUSSION THREAD

**Discussion by KashyapWadhwana** [jrowberg/i2cdevlib#598] [Q&A]
I have successfully uploaded the code on the Arduino Uno and it is showing proper output in the Serial Monitor of the Arduino IDE. 
But when I am uploading the Processing IDE Example in the: i2cdevlib/Arduino/MPU6050/examples/MPU6050_DMP6/Processing/MPUTeapot/
It is not showing any result in the Processing even when I am moving the MPU..
Can u help asap...
Thanks in advance.

**Comment by XuhuaHuang** [comment]
Could you check whether this line: "// #define OUTPUT_TEAPOT" is UNCOMMENTTED? Around line 120 ish I believe
